src/main/java/org/eclipse/lyo/validation/shacl/ShaclShapeFactory.java

```diff
@@ -106,6 +106,8 @@ private static ShaclShape createShaclShape(final Class<?> resourceClass,
         // annotations.
         if (!shaclShape.isReadShaclAnnotations()) {
             shaclShape.setShaclProperties(null);
+            shaclShape.setTargetClass(new URI(resourceClass.getAnnotation(OslcNamespace.class).value()
```

Comment:
how come no unit tests had to be changed?

Reply:
There wasn't a unit test covering this. That's why this bug was left out. Now, we have an example in ValidationExample.java covering this.